### PR TITLE
Nil the actualBlock using the ivar instead of setter in dealloc

### PR DESCRIPTION
**Pull request by chrismaddern**

### CODE REVIEW

**Comment by orta** [comment]
The switch to ARC should remove the need for this

**Comment by chrismaddern** [comment]
But.. then there won't be a little Chris face on the list of contributors...

**Comment by modocache** [comment]
Haha, that seems like a pretty good reason to merge this to me! :+1:

Whaddya say, @orta?

**Comment by chrismaddern** [comment]
cummon @orta!

**Comment by orta** [comment]
hah

File: src/EXPExpect.m [hunk 1]
```diff
@@ -42,7 +42,7 @@ - (id)initWithActualBlock:(id)actualBlock testCase:(id)testCase lineNumber:(int)
 
 - (void)dealloc
 {
-  self.actualBlock = nil;
+  _actualBlock = nil;
```
Comment (supermarin) [inline src/EXPExpect.m]: :+1:

**Pull request merged**